azure-devops/azext_devops/dev/team/_help.py

```diff
@@ -87,6 +87,23 @@ def load_team_help():
     short-summary: Manage security namespaces
     """
 
+    helps['devops security permission update'] = """
+    type: group
+    short-summary: Assign allow or deny permission to given user/group.
+                  Learn more at https://aka.ms/azure-devops-cli-security-permission
```

Comment:
IMO learn more should go in long summary. 

e.g. I would not want to show that level of details in this command: 
```
 D:\>az devops security permission -h

Group
    az devops security permission : Manage security permissions.
        This command group is in preview. It may be changed/removed in a future release.
Subgroups:
    namespace : Manage security namespaces.

Commands:
    list      : List tokens for given user/group and namespace.
    reset     : Reset permission for given permission bit(s).
    reset-all : Clear all permissions of this token for a user/group.
    show      : Show permissions for given token, namespace and user/group.
    update    : Assign allow or deny permission to given user/group.
```

Reply:
Good catch! I agree, it will make more sense if we include the link in the long summary.
 @geverghe what do you think  ?

Reply:
I am not sure of how the long summary appears. @atbagga / @ishitam8 any prior examples where long summary has been used to get a sense of how it would look?

Reply:
Ok got it; makes perfect sense. Including the reference I got from cognitive services that follows the same structure:

`long-summary: This article lists the Azure CLI commands for Azure Cognitive Services account and subscription management only. Refer to the documentation at https://docs.microsoft.com/azure/cognitive-services/ for individual services to learn how to use the APIs and supported SDKs.`

![image](https://user-images.githubusercontent.com/16950956/66916762-89f75280-f039-11e9-8481-d886a94ae7c8.png)

Reply:
![1](https://user-images.githubusercontent.com/43136080/66917035-33d6df00-f03a-11e9-88ea-16cf32c1e12b.PNG)

This is how it looks if added in long-summary with example

Reply:
Raised a fix in Knack for the preview message text to start in new line.
https://github.com/microsoft/knack/pull/167